assemble/Zoltan_integration.F90

```diff
@@ -2130,6 +2122,15 @@ subroutine update_particle_list_element(detector_list_array)
           deallocate(ndets_being_sent)
           cycle
        end if
+       !Check if any are being sent
+       do i = 1, size(ndets_being_sent)
+          if (ndets_being_sent(i)>0) then
+             call mpi_bcast(detector_list%total_attributes, 3, getPINTEGER(), i-1, MPI_COMM_FEMTOOLS, ierr)
```

Comment:
Why do we need to communicate this? Do other processes just get their `%total_attributes` overwritten?

Reply:
This is a relic from when attribute_size was being sent. Back then it was needed to give the size to processors which may have not contained particles beforehand, but with the size stored on detector_lists this is no longer necessary.